fuzz/scripts/run-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
+
+# Number of seconds to run the script. Default to 5 minutes.
+duration="${1:-300}"
+start_time=$SECONDS
+elapsed_time=0
+
+while (( $elapsed_time < $duration )); do
+    for target in $(cargo fuzz_list); do cargo fuzz_run $target --release -- -max_total_time=60 -verbosity=0; done
+    
+    elapsed_time=$(($SECONDS - $start_time))
+done
```

Comment:
I think overall, I would get rid of this script. It seems a little weird to be honest. I can't imagine it being used for anything really.